larisa.py

```diff
@@ -8,59 +8,82 @@
 import requests
 from geopy import Nominatim
 
-def weather_forecast(city):
+
+def temperature_conversion(temp):
+    '''
+    #Author: Nils Becker
+    Converts a given temperature in °K into a °C String representation
+    :param temp: Float Value of temperature in °K
+    :returns: The temperature converted to °C as a String
+    '''
+    return f"{int(temp)-273}°C"

```

Comment:
Should it be `int(temp) - 273` and also... who checks if `temp` is Non a number?